src/main/kotlin/com/github/mckernant1/lolapi/tournaments/TournamentClient.kt

```diff
@@ -46,6 +47,13 @@ class TournamentClient(
         }.also { println(it) }
     }
 
+    fun getStandingsForLeagueByName(leagueName: String, splitYear: Int, splitNumber: Int? = null): List<Standing>  {
+        val leagueClient = LeagueClient()
+        val leagueId = leagueClient.getLeagueByName(leagueName).id
+        print(leagueId)
```

Comment:
Remove prints.
use the the config to print if you want logging

Reply:
noh